tasks/ext.rake

```diff
@@ -75,37 +68,38 @@ task :cross do
   ENV['CXX'] = "#{host}-g++"
 end
 
-file "#{tmp_arch}/stage/lib/libtag.dll" => [install_dll] do |f|
+file "#{Taglib.tmp_arch}/stage/lib/libtag.dll" => [install_dll] do |f|
   install install_dll, f
 end
 
-file install_dll => ["#{tmp}/#{taglib}"] do
-  chdir "#{tmp}/#{taglib}" do
-    sh %(cmake -DCMAKE_INSTALL_PREFIX=#{install_dir} -DCMAKE_TOOLCHAIN_FILE=#{toolchain_file} #{taglib_options})
+file install_dll => ["#{Taglib.tmp}/#{Taglib.dir}"] do
+  chdir "#{Taglib.tmp}/#{taglib}" do
+    sh %(cmake -DCMAKE_INSTALL_PREFIX=#{Taglib.install_dir} -DCMAKE_TOOLCHAIN_FILE=#{toolchain_file} #{taglib_options})
     sh 'make VERBOSE=1'
     sh 'make install'
   end
 end
 
-task vendor: [install_so]
+task vendor: [Taglib.library]
 
-file install_so => ["#{tmp_arch}/#{taglib}", "#{tmp_arch}/#{taglib}-build", "#{tmp}/#{taglib}"] do
-  chdir "#{tmp_arch}/#{taglib}-build" do
-    sh %(cmake -DCMAKE_INSTALL_PREFIX=#{install_dir} #{taglib_options} #{tmp}/#{taglib})
+file Taglib.library => ["#{Taglib.tmp_arch}/#{Taglib.dir}", "#{Taglib.tmp_arch}/#{Taglib.dir}-build", "#{Taglib.tmp}/#{Taglib.dir}"] do
+  chdir "#{Taglib.tmp_arch}/#{Taglib.dir}-build" do
+    sh %(cmake -DCMAKE_INSTALL_PREFIX=#{Taglib.install_dir} #{taglib_options} #{Taglib.source})
+    sh 'make -j 4'
```

Comment:
I added this step to speed up compilation on multi-core machines. I don't know if `make install` is parallel-friendly? Otherwise I could use `make install -j 4`.